src/modules/Api.js

```diff
@@ -0,0 +1,216 @@
+        },
+        {
+          idCategory: '13',
+          strCategory: 'Breakfast',
+          strCategoryThumb:
+            'https://www.themealdb.com/images/category/breakfast.png',
+          strCategoryDescription:
+            'Breakfast is the first meal of a day. The word in English refers to breaking the fasting period of the previous night. There is a strong likelihood for one or more "typical", or "traditional", breakfast menus to exist in most places, but their composition varies widely from place to place, and has varied over time, so that globally a very wide range of preparations and ingredients are now associated with breakfast.',
+        },
+        {
+          idCategory: '14',
+          strCategory: 'Goat',
+          strCategoryThumb:
+            'https://www.themealdb.com/images/category/goat.png',
+          strCategoryDescription:
+            'The domestic goat or simply goat (Capra aegagrus hircus) is a subspecies of C. aegagrus domesticated from the wild goat of Southwest Asia and Eastern Europe. The goat is a member of the animal family Bovidae and the subfamily Caprinae, meaning it is closely related to the sheep. There are over 300 distinct breeds of goat. Goats are one of the oldest domesticated species of animal, and have been used for milk, meat, fur and skins across much of the world. Milk from goats is often turned into goat cheese.',
+        },
+      ],
+    };
+    Counter(DummyData);
+  };
+
+  GetMeals = async () => {
+    await fetch(`${this.FreeMealEP}/1/categories.php`)
+      .then((response) => response.json())
+      .then((json) => {
+        DisplayCards(json.categories);
+        this.GetStats();
+        const comment = document.querySelectorAll('.comment');
+        comment.forEach((item) => {
+          item.addEventListener('click', () => {
+            modal.classList.toggle('hide');
+            this.GetMealInfos(item.id);
+          });
+        });
+      });
+  };
+
+  AddComment = async (data) => {
+    await fetch(
+      `${this.InvolvementApiEP}apps/${this.InvolvementAppID}/comments`,
+      {
+        method: 'POST',
+        body: JSON.stringify(data),
+        headers: {
+          'Content-type': 'application/json; charset=UTF-8',
+        },
+      },
+    );
+    this.DisplayComm(data.item_id);
+  };
+
+  DisplayComm = async (data) => {
+    await fetch(
+      `${this.InvolvementApiEP}apps/${this.InvolvementAppID}/comments?item_id=${data}`,
+    )
+      .then((response) => response.json())
+      .then((json) => {
+        DisplayComments(json);
+      });
+  };
```

Comment:
- [ ] there is an error when the item has no comment yet ! and its because the `json` is not returned as array , that will cause error  in https://github.com/ichala/Javascript-Capstone/blob/7cbe40c787df3082f0dbca6ec5b76a07d05e3ab2/src/modules/Functions.js#L105-L110

so kindly fix that so you get rid of that problem 
suggestion : 
_you can add conditional statement when you get json as response and check if its an array with a minimum length of 1_